docs/HTTP3.md

```diff
@@ -194,6 +191,10 @@ Build curl:
 
 # msh3 (msquic) version
 
+**Note**: The msquic HTTP/3 backend is immature and is not properly functional
+one as of September 2023. Feel free to help us test it and improve it, but
+there is no point in filing bugs about it just yet.
```

Comment:
@bagder What exactly is broken? AFAIK everything should work. If there is something broken, let me know.

Reply:
We have a several http tests disabled with msh3 (`grep msh3 tests/http/*py`) , and a plain `curl --http3 https://curl.se` command line takes too long, which I suspect is an incorrect set of what socket to wait for or what activity to wait for. Possibly more notable when trying the event based API: with a debug build you can run `curl --test-event --http3 https://curl.se/` for that.